kitchen-opennebula.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency 'test-kitchen', '~> 1.2'
   spec.add_dependency 'fog', '~> 1.30'
-  spec.add_dependency 'opennebula', '~> 4.10', '< 5'
+  spec.add_dependency 'opennebula'
```

Comment:
Can we leave '>= 4.10' here, this was never tested with lower versions.

Reply:
Yeah.

Reply:
fixed